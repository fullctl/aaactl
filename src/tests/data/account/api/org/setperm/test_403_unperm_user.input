{
  "client": "api_client_unperm",
  "data": {
    "id": 4,
    "component": "billing.status",
    "permissions": "crud"
  }
}
