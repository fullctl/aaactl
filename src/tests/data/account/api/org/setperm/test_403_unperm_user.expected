{
  "status": 403,
  "response": ""
}
